admin-base/ui.apps/src/main/js/apiImpl.js

```diff
@@ -693,7 +693,10 @@ class PerAdminImpl {
 
       updateWithForm('/admin/updateResource.json' + path + node.path, formData)
       // .then( (data) => this.populateNodesForBrowser(parentPath) )
-      .then(() => resolve())
+      .then(() => resolve()).catch( error => {
```

Comment:
Put the `catch` into a new line please.
It has the same "priority" as `then`

Reply:
done